### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'some')

### ISSUE THREAD

**Issue by igorlesnenko** [ParabolInc/parabol#5902]
Happened 6 times for 2 users in 3 days.

https://sentry.io/organizations/parabol/issues/2936475353/?referrer=github_plugin

```
TypeError: Cannot read properties of undefined (reading 'some')
  at None (./packages/client/components/GroupingKanban.tsx:86:37)
  at Array.forEach (<anonymous>)
  at a (./packages/client/components/GroupingKanban.tsx:82:22)
  at c (./packages/client/components/GroupingKanban.tsx:79:42)
...
(14 additional frame(s) were not displayed)
```

**Comment by Dschoordsch** [comment]
Issue seems to be old since reflections being null was already addressed in a different location in #4354

**Comment by Dschoordsch** [comment]
#6009 might be related, although I could only reproduce this locally on master. It might be that relay has different debug settings in dev and prod, I have to check.
Otherwise I could not reproduce this issue. Looking at the traces 4 of 6 people switched phases back to grouping before it happened. Some had opened spotlight. Nothing strikes me as odd in the logs.
